static/app/views/replays/detail/tagPanel/index.tsx

```diff
@@ -37,26 +37,58 @@ const notTags = [
   'user.id',
   'user.ip',
 ];
+const notSearchable = [
+  'sdk.blockAllMedia',
+  'sdk.errorSampleRate ',
+  'sdk.maskAllInputs',
+  'sdk.maskAllText',
+  'sdk.networkCaptureBodies',
+  'sdk.networkDetailHasUrls',
+  'sdk.networkRequestHasHeaders',
+  'sdk.networkResponseHasHeaders',
+  'sdk.sessionSampleRate',
+  'sdk.useCompression',
+  'sdk.useCompressionOption',
```

Comment:
Should we make `sdk.*` unsearchable in case new options get added?

Reply:
Do you mean searchable from within the index page?

We do have `sdk.name` and `sdk.version` which are searchable from there. Those two are linked in the Tags view to make those searches easier. The new sdk.* things are not linked, they cannot be searched for on the index page.

Within the Tags tab the filters will apply to all the fields.

Reply:
Makes sense, disregard